host-controller/src/main/java/org/jboss/as/host/controller/mgmt/HostControllerRegistrationHandler.java

```diff
@@ -587,13 +592,13 @@ void completeRegistration(final ManagementRequestContext<RegistrationContext> re
             completeTask(!failed);
         }
 
-        void failed(SlaveRegistrationException.ErrorCode error, String message) {
+        void failed(final Throwable t, SlaveRegistrationException.ErrorCode error, String message) {
```

Comment:
Could you add a Java doc describing the argument cannot be null?

Reply:
Fixed.